### PR TITLE
Decaffeinate Tests

### PR DESCRIPTION
**Pull request by icirellik**
**Description**

There is a movement to decaffeinate our legacy javascript to increase developer productivity amongst other tooling benefits. This is the first pass and decaffeinates all of the unit tests that were written in CoffeeScript.

https://artsyproduct.atlassian.net/browse/PLATFORM-2698

### CODE REVIEW

**Comment by damassi** [comment]
@icirellik - not to block in any way, but wondering if you might provide some reasoning behind this change in the PR description? I would estimate that between 95 and 99% of the dev team's work takes place in the `src/v2` folder (TypeScript), and that the the areas of our codebase outside of `v2` that are frequently touched have already been converted to TypeScript. Code that is CoffeeScript is very rarely touched, and only on a need-to basis.

File: src/desktop/apps/about/test/client/router.test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ * DS102: Remove unnecessary code created because of implicit returns
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+const _ = require("underscore")
+const benv = require("benv")
+const rewire = require("rewire")
+const sinon = require("sinon")
+const Backbone = require("backbone")
+
+describe("AboutRouter", function () {
```
Comment (damassi) [inline src/desktop/apps/about/test/client/router.test.js:12]: One meta comment: Is it possible to run decaffinate with a preference towards arrow functions? In day-to-day JS dev one pretty rarely sees `this`, and I worry that in that in the future `this` will be understood even less than it is now.

**Comment by icirellik** [comment]
@damassi definitely! I will also link to the epic.

File: src/desktop/apps/about/test/client/router.test.js (hunk 1 above)
Comment (icirellik) [reply src/desktop/apps/about/test/client/router.test.js:12]: I believe that it is. However, in our CoffeeScript almost every file I've looked at use `this` or variables that begin with `@`. One of the goals in decaffeinating is to enable us to leverage more modern tooling, some of which doesn't support `.coffee`, so modernizing the code wasn't a high priority.

**Pull request merged**